scripts/generate-clients/copy-to-clients.js

```diff
@@ -25,6 +32,28 @@ const getOverwritablePredicate = packageName => pathName => {
   );
 };
 
+const mergeManifest = (fromContent, toContent) => {
+  const merged = {};
+  const toNames = Object.keys(toContent);
+  for (const name of toNames) {
+    if (!fromContent[name]) {
+      merged[name] = toContent[name];
+      continue;
+    }
+    if (typeof toContent[name] === "object") {
```

Comment:
```suggestion
    else if (typeof toContent[name] === "object") {
```

Reply:
not required as if block was removed in https://github.com/aws/aws-sdk-js-v3/pull/900/commits/2ae427a4373225ca7637c146a568f5136b4473ea